### ISSUE TITLE
[Enhancement] Add "Real-Debrid Added" and/or "Added on" dates to sortable columns

### ISSUE THREAD

**Issue by Mahava** [rogerfar/rdt-client#470]
On the Torrents tab to sort by date.

Please and thank you,
-M

**Comment by BlueBull010** [comment]
Yes! Please, this. That would be awesome. I wanted to make a feature request myself but saw this was already asked, which was to be expected. 

When you have many downloads in your list, which tends to happen with torrents if you download a lot, because torrents generally take longer to download than usenet or others, it's sometimes hard to find that torrent you're looking for. It would also make it easier to delete those that will likely not succeed because you just need to delete the oldest ones! Though if I may add ssomething: don't only show the date but also the time (or don't show the time but to keep it behind the scenes and use it to sort) because if you add a lot of torrents at once, sorting by date would help but sorting by date + time would be even better

This would be a very cool addition

**Comment by Cucumberrbob** [comment]
Implemented in #678 included in v2.0.97
